<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GhostDialogUI</class>
 <widget class="QWidget" name="GhostDialogUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>197</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Ghost Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="QGroupBox" name="typeBox">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>120</height>
        </size>
       </property>
       <property name="title">
        <string>Type</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="horizontalSpacing">
           <number>6</number>
          </property>
          <property name="verticalSpacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QRadioButton" name="noneRadio">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>None</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">typeGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="prevRadio">
            <property name="text">
             <string>Previous</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">typeGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="indexRadio">
            <property name="text">
             <string>Indexed</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">typeGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QRadioButton" name="externRadio">
            <property name="text">
             <string>External</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">typeGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="posBox">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="title">
        <string>Position</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QRadioButton" name="backRadio">
            <property name="text">
             <string>Back</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">posGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="frontRadio">
            <property name="text">
             <string>Front</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">posGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="indexBox">
       <property name="title">
        <string>Index</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>8</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="texLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QPushButton" name="texButton">
            <property name="maximumSize">
             <size>
              <width>52</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Tex</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="texLabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="atlasLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QPushButton" name="atlasButton">
            <property name="maximumSize">
             <size>
              <width>52</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Atlas</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="atlasLabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="cellLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QPushButton" name="cellButton">
            <property name="maximumSize">
             <size>
              <width>52</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Cell</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="cellLabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="posGroup"/>
  <buttongroup name="typeGroup"/>
 </buttongroups>
</ui>
